extensions/blocks/podcast-player/icons.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Internal dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+function createSVGs() {
+	// Prevent repeated initialization.
+	if ( document.getElementById( 'jetpack-player-podcast-icons' ) ) {
```

Comment:
This `id` name is a bit inconsistent with others - could we use e.g. `#podcast-player-icons`?

Reply:
I think we might actually be inconsistent with other blocks here by not prefixing with `jetpack`. I haven't looked at everything but it seems at least very common to style blocks with `.jetpack-` classes. 

What about making a separate task to clean this and figure out there whether to add jetpack everywhere or remove it everywhere consistently?

Reply:
Sounds good to me! 👍

Reply:
Todo noted!